api/nodes/serializers.py

```diff
@@ -1466,7 +1458,7 @@ def to_internal_value(self, registration_schema_id):
         return {'registration_schema': schema}
 
 
-class DraftRegistrationSerializer(JSONAPISerializer):
+class DraftRegistrationSerializerLegacy(JSONAPISerializer):
```

Comment:
Change to `DraftRegistrationLegacySerializer`

Reply:
🆗